examples/discovery-search-app/cypress/integration/basic/spec.ts

```diff
@@ -21,18 +21,18 @@ describe('Basic search', () => {
   // Rendering initial page
   describe('When the example app loads', () => {
     it('SearchInput has placeholder text "Search"', () => {
-      cy.get('.bx--search-input').should('have.attr', 'placeholder', 'Search');
+      cy.findByPlaceholderText('Search').should('have.attr', 'placeholder', 'Search');
     });
 
     it('SearchInput has magnifying glass icon', () => {
-      cy.get('.bx--search-magnifier').should('be.visible');
+      cy.get('svg.bx--search-magnifier').should('be.visible');
     });
   });
 
   // Querying with results
   describe('When entering a query with results', () => {
     beforeEach(() => {
-      cy.get('.bx--search-input').type('abil{enter}');
+      cy.findByPlaceholderText('Search').type('abil{enter}');
```

Comment:
Does it make sense to add an alias for `@searchInput` in this file too for consistency?

Reply:
Yeah, I'm down with that